docs/content/blog/v1-alpha3.md

```diff
@@ -0,0 +1,152 @@
+
+```toml
+default-storage = "mydb"
+
+[[storage]]
+  name = "mydb"
+  plugin = "mongodb"
+
+  [storage.config]
+    url = "mongodb://username:password@host:port"
+```
+
+CosmosDB works as well, though we have a known performance problem with our CosmosDB indices that will be addressed in the next release.
+
+Any existing v0.38 compatible storage mixins, such as the azure storage plugin, do not work with this release.
+We are still finalizing the new storage plugin protocol, so we suggest holding off on making a custom storage plugin until its ready.
+
+## Namespaces
+Porter resources such as installations, credential sets and parameter sets can optionally be defined within a namespace. 
+Resources that are not defined in a namespace are considered global.
+When an installation is defined in a namespace, it can reference a credential or parameter set that is also defined in that namespace or at the global scope.
+Resources defined globally cannot reference other resources that are defined in a namespace.
+
+You can set the current namespace in the [Porter configuration file](https://release-v1.porter.sh/configuration/#config-file) using the namespace setting.
+
+When an installation references a parameter or credential set, Porter first looks for a resource with that name in the current namespace.
+If one does not exist, Porter then looks for that resource at the global level.
+This lets you define a common set of credentials to be used for an environment, like staging environment credentials that everyone can reuse, while allowing for overriding that resource within a particular namespace.
+
+```bash
+# list just in the current namespace defined in your porter config
+porter installations list
+
+# list in the specified namespace
+porter installation list --namespace myuser
+
+# list across all namespaces
+porter installation list --all-namespaces
+
+# list across all namespaces (alternate syntax)
+porter installation list --namespace '*'
+```
+
+## Labels
+Porter resources also now support labels which may be used for filtering with the list commands.
+For example:
+
+```
+porter installations list --label env=dev --label app=myapp
+porter credentials list --label env=dev
+porter parameters list --label env=dev
+```
+
+## Bundle State
+Bundle authors can now declare that certain files are "state" and should be preserved between bundle runs.
+A prime example is the tfstate and tfvars files created by the terraform mixin.
+Now instead of using the old method of declaring those files as both parameters and outputs, you should declare them as **state** and Porter will ensure that they are always available to your bundle.
+
+This corrects the problem where if an installation that used the terraform mixin, without a remote backend configured, failed to install, on subsequent install runs the resources originally created by terraform were not recorded and reused, leaving it up to the user to manually cleanup resources.
```

Comment:
nit: I wonder if this sentence might be revised a bit for potential readability improvements.

Maybe something along the lines of "This corrects the problem encountered by installations using the terraform mixin wherein resources created by terraform were not recorded and reused after a failed install, leaving it up to the user to manually clean up resources."?

Reply:
Thanks this reads much better than what I wrote! 👍